nix/default.nix

```diff
@@ -91,13 +91,11 @@ let
 
         # Rust nightly
         (self: super: let
-          rust-channel = self.moz_overlay.rustChannelOf { date = "2023-04-21"; channel = "nightly"; };
+          rust-channel = self.moz_overlay.rustChannelOf { date = "2024-07-28"; channel = "nightly"; };
         in rec {
           rustc-nightly = rust-channel.rust.override {
             targets = [
-               "wasm32-unknown-emscripten"
```

Comment:
oh, why were these here?

Reply:
We used emscripten to get PIC-relocatable shared Wasm modules for the RTS. This is now redundant by the customized RTS backends.